rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -318,7 +323,7 @@ void getNewFederation_initialVersion() {
             RskAddress contractAddress = invocation.getArgument(0);
             DataWord address = invocation.getArgument(1);
 
-            assertArrayEquals(Hex.decode("aabbccdd"), contractAddress.getBytes());
+            assertArrayEquals(bridgeAddressBytes, contractAddress.getBytes());
```

Comment:
before, the bridge storage provider was being created [with a mock "aabbccdd" address](https://github.com/rsksmart/rskj/blob/master/rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java#L321). 
But now we are using the federation storage provider, that is always created with the bridge address. That's why now the assertion should check if the contractAddress is the real bridge address.

Reply:
Why compare bytes? You could simply compare RskAddress objects